Software/tft/header/config_test.go

```diff
@@ -0,0 +1,64 @@
+package header
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestConfigCheck(t *testing.T) {
+	c := &Config{
+		Apply:            false,
+		InputPath:        "",
+		OutputPath:       "",
+		VendorId:         0,
+		DeviceId:         0,
+		HardwareRevision: -1,
+		InputFile:        nil,
+		OutputFile:       nil,
+	}
+
+	err := CheckConfig(c)
+	require.Error(t, err, "expect error on empty input filename")
```

Comment:
Just FYI, the comment is optional

Reply:
Yeah, I know, that was intended to provide our expectations for config